restapi/src/main/java/io/stargate/web/docsapi/service/DocsShredder.java

```diff
@@ -1,8 +1,8 @@
 package io.stargate.web.docsapi.service;
 
+import com.datastax.oss.driver.shaded.guava.common.base.Splitter;
 import com.fasterxml.jackson.databind.JsonNode;
 import com.fasterxml.jackson.databind.node.ValueNode;
-import com.google.common.base.Splitter;
```

Comment:
Was Eric faster in merging the shredding than you? Can we ensure once we merge this that it's top of the master and there are no direct usages of guava?

Reply:
I am not 100% sure how this came about. I will make sure to merge from master and check that the only Guava usage is from `persistence-cassandra-4.0/src/main/java/io/stargate/db/cassandra/impl/StargateSystemKeyspace.java` (which is due to Cassandra leaking `ListenableFuture`)